contrib/outmessage-irccolors.tengo

```diff
@@ -0,0 +1,7 @@
+// See https://github.com/42wim/matterbridge/issues/798
+
+// if we're not sending to an irc bridge we strip the IRC colors
```

Comment:
Should we be using all the `contrib/*.tengo` files by default? imo this would still be a bug with default configuration

Reply:
The irccolors is already enabled by default, see https://github.com/42wim/matterbridge/blob/master/internal/tengo/outmessage.tengo which is also in this PR

The other contrib tengo files are just examples.

Or do I misunderstand you and is there another bug you're talking about?

Reply:
Oh right! My bad